src/helpers/_styleguide-example.scss

```diff
@@ -0,0 +1,80 @@
+/*
+ * _styleguide-example.scss
+ *
+ * This is an example file that adheres to our .stylelintrc rules and style.
+ */
+
+@import url('x.css');
+@import url('y.css');
+
+$sass-variable: 10px;
+
+/*
+ * Multi-line comment
+ */
+.base-1,
+.base-2,
+.base-3__element[type="text"] {
+  background: linear-gradient(#fff, rgba(0, 0, 0, 0.8));
+  box-sizing: border-box;
+  display: block;
+  color: #333;
+}
+
+.base-1__element,
+.base-2__element:not(:first-child) {
+  padding: 10px;
+  top: calc(calc(1em * 2) / 3);
+}
+
+.base-1__element { width: 10%; }
+.base-2__element { width: 20%; }
+.base-3__element { width: 30%; }
+
+/* Single-line comment */
```

Comment:
Why are we doing vanilla CSS comments instead of SCSS comments, e.g. `// this is a comment`?